public/template/lib/layout.pug

```diff
@@ -56,9 +56,9 @@ html(lang="en")
             b #{config.site} 
             | brought to you by 
             if config.ownerURL
-              a(href="#{config.ownerURL}") #{config.owner}
+              a(href=config.ownerURL) #{config.owner}
             else
-              | #{config.owner}
+              | config.owner
```

Comment:
Won't this just insert the literal text `config.owner`?

Reply:
done https://github.com/pump-io/pump.io/pull/1580/commits/b61e2b4aa8d52f8eb584ef6b47389000d848e0e2